app/gui2/src/components/GraphEditor.vue

```diff
@@ -484,7 +528,7 @@ function handleNodeOutputPortDoubleClick(id: ExprId) {
       :modes="EXECUTION_MODES"
       :breadcrumbs="breadcrumbs"
       @breadcrumbClick="console.log(`breadcrumb #${$event + 1} clicked.`)"
-      @back="console.log('breadcrumbs \'back\' button clicked.')"
+      @back="projectStore.executionContext.pop()"
```

Comment:
Is `graphStore.updateState()` needed in `exitNode` but not here?

Reply:
Good point, this should use the `leaveNode` function.